CMSgov/design-system
# [WNMGDS-452] [WNMGDS-454] Update Sketch grid and fix font used in vertical nav

Updated the grid in the Sketch file to be consistent with code for XL and L (Desktop sizing)
![Screen Shot 2020-05-11 at 2 41 10 PM](https://user-images.githubusercontent.com/1449852/81600516-bee14300-9397-11ea-8e4e-c448ba864a1b.png)
![Screen Shot 2020-05-11 at 2 41 28 PM](https://user-images.githubusercontent.com/1449852/81600530-c4d72400-9397-11ea-87c4-652ef766d403.png)
![Screen Shot 2020-05-11 at 2 41 15 PM](https://user-images.githubusercontent.com/1449852/81600547-cb659b80-9397-11ea-8d73-17806b42d826.png)

![Screen Shot 2020-05-11 at 2 41 41 PM](https://user-images.githubusercontent.com/1449852/81600664-fe0f9400-9397-11ea-9090-bc78f7a6a7f6.png)


Fixed the font for the active vertical nav item to be Open Sans
<img width="314" alt="Screen Shot 2020-05-11 at 2 55 49 PM" src="https://user-images.githubusercontent.com/1449852/81600628-e9330080-9397-11ea-9e35-c6e3f0349f4e.png">



Closes #344 
Closes #678 


**How to test**
1. Download and open up Sketch file https://github.com/CMSgov/design-system/blob/WNMGDS-452-Sketch-grid/design-assets/CMS-Design-System-UI-kit.sketch
1. Look at the Layout artboards under layers to verify the XL and LG layouts are there are set up properly
1. Next, click on components under layers and go to the vertical navigation component. Ensure that the proper font, Open Sans, is being used for the current page links
